front/app/api/navbar/useAddNavbarItem.ts

```diff
@@ -10,23 +10,33 @@ import navbarKeys from './keys';
 import { INavbarItemResponse, INavbarItemAdd } from './types';
 
 const addNavbarItem = (item: IItemNotInNavbar) => {
-  const navbarItem: INavbarItemAdd =
-    item.type === 'default_item'
-      ? {
-          code: item.navbarCode,
-          title_multiloc: item.titleMultiloc,
-        }
-      : 'pageId' in item
-      ? {
-          code: 'custom',
-          static_page_id: item.pageId, // static page
-          title_multiloc: item.titleMultiloc,
-        }
-      : {
-          code: 'custom',
-          project_id: item.projectId, // project
-          title_multiloc: item.titleMultiloc,
-        };
+  let navbarItem: INavbarItemAdd;
+
+  if (item.type === 'default_item') {
+    navbarItem = {
+      code: item.navbarCode,
+      title_multiloc: item.titleMultiloc,
+    };
+  } else if ('pageId' in item) {
+    navbarItem = {
+      code: 'custom',
+      static_page_id: item.pageId,
+      title_multiloc: item.titleMultiloc,
+    };
+  } else if (item.type === 'project') {
+    navbarItem = {
+      code: 'custom',
+      project_id: item.itemId,
+      title_multiloc: item.titleMultiloc,
+    };
+  } else {
```

Comment:
Maybe add a comment that this last item means it's a folder. It's not clear from the code.

Reply:
Done.